src/components/victory-label.jsx

```diff
@@ -1,35 +1,156 @@
+export default class VictoryLabel extends React.Component {
+  static propTypes = {
+    /**
+     * The capHeight prop defines a text metric for the font being used: the
+     * expected height of capital letters. This is necessary because of SVG,
+     * which (a) positions the *bottom* of the text at `y`, and (b) has no
+     * notion of line height. The value should ideally use the same units as
+     * `lineHeight` and `dy`, preferably ems. If given a unitless number, it
+     * is assumed to be ems.
+     */
+    capHeight: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number
+    ]),
+    /**
+     * The children of this component define the content of the label. This
+     * makes using the component similar to normal HTML spans or labels.
+     * Currently, only strings are supported.
+     */
+    children: PropTypes.string, // TODO: Expand child support in future release
+    /**
+     * The lineHeight prop defines how much space a single line of text should
+     * take up. Note that SVG has no notion of line-height, so the positioning
+     * may differ slightly from what you would expect with CSS. The value
+     * should ideally use the same units as `capHeight` and `dy`, preferably
+     * ems. If given a unitless number, it is assumed to be ems.
+     */
+    lineHeight: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number
+    ]),
+    /**
+     * The style prop applies CSS properties to the rendered `<text>` element.
+     */
+    style: PropTypes.object,
+    /**
+     * The textAnchor prop defines how the text is horizontally positioned
+     * relative to the given `x` and `y` coordinates.
+     */
+    textAnchor: PropTypes.oneOf([
+      "start",
+      "middle",
+      "end",
+      "inherit"
+    ]),
+    /**
+     * The verticalAnchor prop defines how the text is vertically positioned
+     * relative to the given `x` and `y` coordinates.
+     */
+    verticalAnchor: PropTypes.oneOf([
+      "start",
+      "middle",
+      "end"
+    ]),
+    /**
+     * The transform prop applies a transform to the rendered `<text>` element.
+     * In addition to being a string, it can be an object containing transform
+     * definitions for easier authoring.
+     */
+    transform: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.object
+    ]),
+    /**
+     * The x prop defines the x coordinate to use as a basis for horizontal
+     * positioning.
+     */
+    x: PropTypes.number,
+    /**
+     * The y prop defines the y coordinate to use as a basis for vertical
+     * positioning.
+     */
+    y: PropTypes.number,
+    /**
+     * The dy prop defines a vertical shift from the `y` coordinate. Since this
+     * component accounts for SVG weirdness and accounts for `capHeight`,
+     * `lineHeight`, and `verticalAnchor`, this will usually not be necessary.
+     */
+    dy: PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.string
+    ])
+  };
```

Comment:
Notably absent: no `standalone` prop. A couple reasons:

1. Seems unnecessary.
2. I really wanted the `style` prop to be only text styles, with no nesting (with `parent` and `text` sub-objects). If we had `standalone`, we'd pretty much force the `style` prop to be a nested object, because you'd need a way to send width, height, and other stuff to the rendered `<svg>` element.